src/collectd.conf.pod

```diff
@@ -3381,6 +3409,11 @@ If a sensor disappears a notification is sent.
 If you have for example dual power supply and one of them is (un)plugged then
 a notification is sent.
 
+=item B<NotifyIPMIConnectionState> I<true>|I<false>
+
+If a IPMI connection state changes after initialization time of a minute
+a notification is sent. Defaults to B<false>.
+
```

Comment:
Please add a note about legacy support, i.e. that any option other than `Instance` will trigger legacy config handling which skipts the `<Instance>` block and that this support will go away in the next major version of collectd.